cmd/utils/flags.go

```diff
@@ -799,6 +799,11 @@ var (
 		Value: true,
 	}
 
+	AstridFlag = cli.BoolFlag{
```

Comment:
let's be a bit more precise and call this `PolygonAstridSync` (or just `PolygonSync`, later on when we are done with the project, the `Astrid` name should go away, because it is a project code name)

same for the setting in eth/ethconfig/config.go